File: app/src/main/java/onefootball/com/testonefootball/framework/PushRegistrationCategory.java
package onefootball.com.testonefootball.framework;

/**
 * Push registration categories to register against to receive pushes for that category.
 */
public enum PushRegistrationCategory
{
    TEAM("it%d");

    private final String value;

    PushRegistrationCategory(String value)
    {
        this.value = value;
    }

    public String getValue()
    {
        return this.value;
    }

    public String toString()
    {
        return this.value;
    }

}

File: app/src/main/java/onefootball/com/testonefootball/framework/PushRegistrationCategory.kt
package onefootball.com.testonefootball.framework

/**
 * Push registration categories to register against to receive pushes for that category.
 */
enum class PushRegistrationCategory(val value: String) {
    TEAM("it%d");

    override fun toString(): String {
        return this.value
    }
}
